apps/site/package.json

```diff
@@ -1,20 +1,6 @@
 {
-  "type": "module",
-  "private": true,
   "name": "@node-core/website",
-  "description": "Nodejs.org Website",
-  "homepage": "https://nodejs.org",
```

Comment:
this key supports the [`npm docs` command](https://docs.npmjs.com/cli/v9/commands/npm-docs)

Reply:
These are already on the root package.json